eng/scripts/Update-Emojis.ps1

```diff
@@ -0,0 +1,50 @@
+<#
+.SYNOPSIS
+Replace emoji :<name>: expressions with \emoji :<name>: syntax that Doxygen's Markdown renderer can use
```

Comment:
Is this really worth the effort? Can we not simply use a char set that both github and Doxygen agree upon?

Reply:
Did you consider simply feeding html files to Doxygen? Does it support that? If so we can use github APIs to convert from MD to html and not need to be a middle man for converting between the 2. We also need to think about other locations like samples where we don't know if these will correctly render or not.

Reply:
Another option is to directly use the unicode chars from http://unicode.org/emoji/charts/full-emoji-list.html. Github supports doing something like `&#x1F600;` which is the same as `:smile:` so assuming Doxygen also supports the unicode char set it should work.

Reply:
The HTML unicode character codes do not work in Doxygen (the literal text gets rendered instead of the character). Doxygen does have support for a subset of [HTML entities](https://www.doxygen.nl/manual/htmlcmds.html). In this case I was able to eliminate the script by using HTML entities that look similar to the emojis. 

@danewalton  see example rendering in the Key Features table at: https://djurekdocs.z5.web.core.windows.net/emoji/md_sdk_docs_iot__r_e_a_d_m_e.html

I could be missing this in the documentation but it seems that adding HTML sections (i.e. rendering the markdown to HTML) to individual pages is not a supported scenario in Doxygen. It wants markdown or a supported file format with doxygen commands.

Reply:
It is possible to insert HTML content into the middle of a markdown file (or any file) using the `\includehtml` ... It's possible to roundtrip the markdown file through the github API and get some HTML and then store the HTML in a separate file and replace the contents of the markdown file with a `\includehtml rendered_html_output_of_this_markdownfile.html` but I'm not sure about the implications of setting the `EXAMPLE_PATH` ... I think the current solution solves the problem but this one might make sense to explore in the future.